shared/test/dataDriver_test.js

```diff
@@ -151,6 +151,19 @@ describe("loop.DataDriver", () => {
     });
   });
 
+  describe("#handleEvent.error", () => {
+    it("should retry the connection on error", () => {
+        let originalConnect = driver._connectToRoom;
+        sandbox.stub(driver, "_connectToRoom", token => {
+          originalConnect.call(driver, token);
+        });
+        driver.handleEvent({ type: "error" });
+        sinon.assert.notCalled(dispatcher.dispatch);
+        clock.tick(driver.INITIAL_RETRY_TIMEOUT + 1);
+        sinon.assert.called(driver._connectToRoom);
```

Comment:
I found a workaround. In the test, I need to call _connectToRoom to set things up properly, but then am able to replace _connectToRoom with a do-nothing stub after things have been set up.